README.md

```diff
@@ -8,10 +8,16 @@ The only configurable option is the log level, which defaults to *INFO*.
 
 This work is covered by the Hippocratic Licence v3, as documented in [LICENSE.md](LICENSE.md).
 
-
 ## Installation
 
-Building the `JAR` file requires both Maven and Java to be installed. The `JAR` file can be built using Maven: `mvn package`
+Building the `JAR` file requires Java to be installed.
```

Comment:
I'd prefer to see this block of the README written differently:

---
Building this project requires Java 17 to be installed, and the use of Maven. If you have Maven v3.9.2 or better available to you locally, then run `mvn package` to compile and generate the package. Alternatively, you can use the Maven wrapper included with this repo.

In Unix-like environments (Linux, Mac OS etc) run

`./mvnw package`

On Windows you can run

`.\mvnw.cmd package`

To find out more about the Maven wrapper, please visit [https://www.baeldung.com/maven-wrapper](https://www.baeldung.com/maven-wrapper)

Reply:
100% can make the requested changes @jmcp.

One thing I would say about suggesting people can use local `mvn` to build the project... one of the main benefits of the maven wrapper is that it helps to enforce building the project with a specific version of Maven.  If someone uses their local version of Maven, it might not be compatible with the version specified in the wrapper.

In such a small project like this, it probably won't have much impact, but I have spent many hours (_perhaps days_) in the past facing build problems just because my build tool version is incorrect.

Having said all this, I will update the Readme as you requested 🤝